TempMail/utils/Email.swift

```diff
@@ -10,26 +10,27 @@
 import SwiftUI
 
 class Email : ObservableObject{
-    @Published var email_addr:String?
+    @Published var email_addr: String
     
-     func GenEmail() {
+//  init method
+    init() {
+        self.email_addr = ""
+        genEmailAddr()
+    }
+//  generates an email address
+    func genEmailAddr() -> Void {
```

Comment:
Since you're using an init, have this function return a string and then set self.email_addr in the init().

Reply:
When the user clicks generate new email, instantiate a new Email object. I think instantiating a new object is preferable to relying on functions with side effects and conforms better to the conceptual model

Reply:
Then the email var of self in ContentView has to be mutable so that it can be assigned a new object every time, correct? That means it should be an @State variable. Is there another approach I'm not aware of, or is it ok to make it @State?